cmd/istioctl/main.go

```diff
@@ -0,0 +1,225 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/golang/glog"
+	"github.com/golang/protobuf/proto"
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/pkg/api"
+
+	"istio.io/manager/cmd"
+	"istio.io/manager/model"
+)
+
+var (
+	// input file name
+	file string
+
+	key    model.Key
+	schema model.ProtoSchema
+
+	postCmd = &cobra.Command{
+		Use:   "post [kind] [name]",
+		Short: "Create a configuration object",
+		RunE: func(c *cobra.Command, args []string) error {
+			if len(args) != 2 {
+				return fmt.Errorf("Provide kind and name")
```

Comment:
nit: lowercase error strings here and elsewhere unless the cobra command error handling doesn't prefix these errors with anything else (see https://github.com/golang/go/wiki/CodeReviewComments#error-strings)